### PR TITLE
Add projects

### PR DESCRIPTION
**Pull request by BeJuanitox**
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/7851ea6f75.js" crossorigin="anonymous"></script>
```
Comment (Andres2D) [inline index.html]: The scripts should be placed at the bottom of the html file, before the closing body.

File: index.html [hunk 2]
```diff
@@ -153,6 +154,71 @@ <h1 class="title">Skills</h1>
...
+      <div class="card">
+        <img class="image" src="./images/projects/CotejoApp.png" alt="CotejoApp">
+        <div class="info">
+          <h2 class="name">CotejoApp</h2>
+          <p class="texts">A funny application to schedule matchs of football.</p>
+        </div>
+        <div class="links">
+          <div class="urls">
+            <a href="https://github.com/Andres2D/cotejo-app-front" target="_blank">
+              <i class="fa-brands fa-github" style="color: black;"></i>
```
Comment (Andres2D) [inline index.html]: Use a class instead of a style attribute.

File: index.html [hunk 3]
```diff
@@ -153,6 +154,71 @@ <h1 class="title">Skills</h1>
...
+            </a>          
+          </div>
+          <div class="urls">
+            <a Class="text-cards" href="https://cotejo-app-beta.netlify.app/login" target="_blank">
+              <i class="fa fa-external-link" aria-hidden="true" style="color: #006AFF;"></i>
```
Comment (Andres2D) [inline index.html]: same here

File: index.html [hunk 4]
```diff
@@ -153,6 +154,71 @@ <h1 class="title">Skills</h1>
...
+      </div>
+      <!-- Form Card -->
+      <div class="card">
+        <img class="image" src="./images/projects/Form.png" alt="Form">
+        <div class="info">
+          <h2 class="name">Form</h2>
+          <p class="texts">Form that adds data to a dynamic table.</p>
+        </div>
+        <div class="links">
+          <div class="urls">
```
Comment (Andres2D) [inline index.html]: This seems to be redundant, you should use a single div with all the <a> tags inside.
Review how to improve it from the css.

File: index.html [hunk 5]
```diff
@@ -153,6 +154,71 @@ <h1 class="title">Skills</h1>
       </div>
     </div>
   </div>
+  <h1 class="title"> Projects </h1>
+  <div class="projects">
+    <div class="bodycards">
```
Comment (Andres2D) [inline index.html]: Why do you need this nested div?.
This should be a single div. Review how to improve the css classes.

File: styles.css [hunk 6]
```diff
@@ -183,3 +184,65 @@ svg {
   margin-top: -98.5%;
   rotate: -90deg;
 }
+
+.bodycards {
```
Comment (Andres2D) [inline styles.css]:
```suggestion
.bodyCards {
```

**Review by Andres2D — CHANGES_REQUESTED** [review]
Try to avoid the style atrributes, use classes instead.

File: index.html (hunk 2 above)
Comment (BeJuanitox) [reply index.html]: I tried but it didn't work

**Review by Andres2D — APPROVED** [review]
👍

**Pull request merged**